cms/templates/course_outline.html

```diff
@@ -57,6 +57,8 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
   </div>
   %endif
 
+## preventing this error from showing up as we support all the xblocks that are installed.
```

Comment:
I don't think this is accurate..

We're not disabling this because we support all XBlocks;
we're disabling this because the feature breaks.

What "we support all the xblocks" has to do w/ this is that it's the context which allows us to use this particular approach.

Please amend as follows:

```python
## This feature intermittently fails for us at Stanford.
## The system mistakenly thinks that _all_ XBlocks are deprecated.
## Since we're unsure what's causing this, it's difficult to test, and we don't actually use the feature,
## we've decided to simply remove it from the page.
```